samples/cellular/nrf_cloud_mqtt_multi_service/src/main.c

```diff
@@ -33,12 +35,34 @@ K_THREAD_DEFINE(msg_thread, CONFIG_MESSAGE_THREAD_STACK_SIZE, message_queue_thre
 K_THREAD_DEFINE(con_thread, CONFIG_CONNECTION_THREAD_STACK_SIZE, cloud_connection_thread_fn,
 		NULL, NULL, NULL, 0, 0, 0);
 
+#if defined(CONFIG_NRF_CLOUD_COAP)
+/* Define, and automatically start the CoAP FOTA check thread. See fota_support_coap.c */
+K_THREAD_DEFINE(coap_fota, CONFIG_COAP_FOTA_THREAD_STACK_SIZE, coap_fota_thread_fn,
+		NULL, NULL, NULL, 0, 0, 0);
+
+/* Define, and automatically start the CoAP shadow check thread. See shadow_support_coap.c */
+K_THREAD_DEFINE(coap_shadow, CONFIG_COAP_SHADOW_THREAD_STACK_SIZE, coap_shadow_thread_fn,
+		NULL, NULL, NULL, 0, 0, 0);
+#endif
+
 /* main() is called from the main thread, which defaults to priority zero,
  * but for illustrative purposes we don't use it. main_application() could be called directly
  * from this function, rather than given its own dedicated thread.
  */
 int main(void)
 {
-	LOG_INF("nRF Cloud MQTT multi-service sample has started, version: %s", CONFIG_APP_VERSION);
+	LOG_INF("nRF Cloud Multi Service sample has started, version: %s",
+		CONFIG_APP_VERSION);
```

Comment:
i don't remember what the objections were to making this one log msg...
so ignore if no one likes this:

```
LOG_INF("nRF Cloud multi-service sample has started, version: %s, protocol: %s",
		CONFIG_APP_VERSION,
		IS_ENABLED(CONFIG_NRF_CLOUD_COAP) ? "CoAP" : "MQTT");
```

Reply:
Georges suggested a second line.  Doesn't matter to me.

Reply:
Being a single or separate lines is not particularly important to me, as long as the sample is named multi-service sample